trigger

```diff
@@ -121,11 +142,11 @@ run
 if [[ $watchall = true ]]; then
     # Watch all files in the current directory
 
-    while cfile=$(inotifywait --quiet --format '%w%f' --event close_write,move_self --exclude '\.git' -r .); do
+    while cfile=$( "${watch_all_cmd[@]}" | head -n1 ); do
```

Comment:
Even with the `-1` flag `fswatch` sometimes still outputs multiple changes at once. It seems sensible enough to just print out one of the changed files.